README.md

```diff
@@ -98,38 +99,52 @@ We offer the following installation methods:
    - kubectl (can be installed via `gcloud components install kubectl`)
    - Local Kubernetes cluster deployment tool:
         - [Minikube (recommended for
-         Linux)](https://kubernetes.io/docs/setup/minikube/).
-        - Docker for Desktop (recommended for Mac/Windows): It provides Kubernetes support as [noted
-     here](https://docs.docker.com/docker-for-mac/kubernetes/).
+         Linux)](https://kubernetes.io/docs/setup/minikube/)
+        - [Docker for Desktop (recommended for Mac/Windows)](https://www.docker.com/products/docker-desktop)
+          - It provides Kubernetes support as [noted
+     here](https://docs.docker.com/docker-for-mac/kubernetes/)
+        - [Kind](https://github.com/kubernetes-sigs/kind)
    - [skaffold]( https://skaffold.dev/docs/install/) (ensure version ≥v0.20)
 
 1. Launch the local Kubernetes cluster with one of the following tools:
 
-    - Launch Minikube (tested with Ubuntu Linux). Please, ensure that the
+    - To launch **Minikube** (tested with Ubuntu Linux). Please, ensure that the
        local Kubernetes cluster has at least:
         - 4 CPU's
         - 4.0 GiB memory
+      ```shell
+      minikube start --cpus=4 --memory 4096
+      ```
 
-        To run a Kubernetes cluster with Minikube using the described configuration, please run:
-
-    ```shell
-    minikube start --cpus=4 --memory 4096
-    ```
-    
-    - Launch “Docker for Desktop” (tested with Mac/Windows). Go to Preferences:
+    - To launch **Docker for Desktop** (tested with Mac/Windows). Go to Preferences:
         - choose “Enable Kubernetes”,
         - set CPUs to at least 3, and Memory to at least 6.0 GiB
         - on the "Disk" tab, set at least 32 GB disk space
 
+    - to Launch a **Kind** cluster:
```

Comment:
would it be worth adding the criteria above as well? (need a VM of this size etc) ?

Reply:
I tried looking for this information, but kind doesn't seem to manage resources in the same way as the others. The [Istio docs](https://istio.io/docs/setup/platform-setup/kind/) don't mention anything about CPU/RAM either, even though it does for [minikube](https://istio.io/docs/setup/platform-setup/minikube/) and [docker for desktop](https://istio.io/docs/setup/platform-setup/docker/), so I decided to follow suit

Let me know if you have a better answer though

Reply:
SGTM. we can skip.